pkg/config/config.go

```diff
@@ -271,6 +287,40 @@ func (c *Config) incorporateUserSettings(u *Config) {
 	if u.Kubelet != nil {
 		c.Kubelet = u.Kubelet
 	}
+
+	if u.Ingress.TuningOptions.HeaderBufferBytes > 0 {
+		c.Ingress.TuningOptions.HeaderBufferBytes = u.Ingress.TuningOptions.HeaderBufferBytes
+	}
+	if u.Ingress.TuningOptions.HeaderBufferMaxRewriteBytes > 0 {
+		c.Ingress.TuningOptions.HeaderBufferMaxRewriteBytes = u.Ingress.TuningOptions.HeaderBufferMaxRewriteBytes
+	}
+	if u.Ingress.TuningOptions.HealthCheckInterval != nil {
+		c.Ingress.TuningOptions.HealthCheckInterval = u.Ingress.TuningOptions.HealthCheckInterval
+	}
+	if u.Ingress.TuningOptions.ClientTimeout != nil {
+		c.Ingress.TuningOptions.ClientTimeout = u.Ingress.TuningOptions.ClientTimeout
+	}
+	if u.Ingress.TuningOptions.ClientFinTimeout != nil {
+		c.Ingress.TuningOptions.ClientFinTimeout = u.Ingress.TuningOptions.ClientFinTimeout
+	}
+	if u.Ingress.TuningOptions.ServerTimeout != nil {
+		c.Ingress.TuningOptions.ServerTimeout = u.Ingress.TuningOptions.ServerTimeout
+	}
+	if u.Ingress.TuningOptions.ServerFinTimeout != nil {
+		c.Ingress.TuningOptions.ServerFinTimeout = u.Ingress.TuningOptions.ServerFinTimeout
+	}
+	if u.Ingress.TuningOptions.TunnelTimeout != nil {
+		c.Ingress.TuningOptions.TunnelTimeout = u.Ingress.TuningOptions.TunnelTimeout
+	}
+	if u.Ingress.TuningOptions.TLSInspectDelay != nil {
+		c.Ingress.TuningOptions.TLSInspectDelay = u.Ingress.TuningOptions.TLSInspectDelay
+	}
```

Comment:
Do we want to follow the same validation pattern as the ingress operator does? https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/operator/controller/ingress/deployment.go#L611-L634